### ISSUE TITLE
Allow include from same/parent directory

### ISSUE THREAD

**Issue by Dexus** [jgm/pandoc#8058]
**Explain the problem.**
We have some docs in rst fileformat but when we try to convert them to markdown. we see warning like
```
[WARNING] Could not load include file '../common/introduction.rst' at line 7 column 1
[WARNING] Could not load include file '/common/introduction.rst' at line 7 column 1
[WARNING] Could not load include file './common/introduction.rst' at line 7 column 1
...
```
I would at least expect that the include is possible from the same folder `./xyz`, but also include from the parrent with `../xyz` but this does not work.

Mabye a "base_path" would nice to limit the "root" for includes like `/common/introduction.rst`.

**Pandoc version?**
Ubuntu 20.04
pandoc 2.5
Compiled with pandoc-types 1.17.5.4, texmath 0.11.2.2, skylighting 0.7.7

**Comment by jgm** [comment]
You've got a very old version; you might have somewhat better results with the latest, 2.18.  Try with that and report back.

**Comment by Dexus** [comment]
Hi @jgm,
with the 2.18 deb the output has changed:

This message is gone:
```
[WARNING] Could not load include file './common/introduction.rst' at line 7 column 1
```

but still exist:
```
[WARNING] Could not load include file '../common/introduction.rst' at line 7 column 1
[WARNING] Could not load include file '/common/introduction.rst' at line 7 column 1
```

**Comment by jgm** [comment]
> would it make sense here to use the current working dir as "base" for /

I don't think so; after all this is a perfectly valid absolute path. Someone might pass in something like `/usr/local/share/foo/bar.rst`.

**Comment by Dexus** [comment]
sphinx does work other, it use the src dir as `/`.

**Comment by jgm** [comment]
If not just sphinx but docutils does this, I'd reconsider.
Keeping this open for now.
